indico/modules/events/papers/schemas.py

```diff
@@ -80,28 +102,64 @@ class PaperRevisionSchema(mm.ModelSchema):
     state = EnumField(PaperRevisionState)
     timeline = PaperRevisionTimelineField()
     judgment_comment_html = Function(lambda revision: escape(revision.judgment_comment))
+    reviewer_data = Method('_get_reviewer_data')
 
     class Meta:
         model = PaperRevision
         fields = ('id', 'submitted_dt', 'judgment_dt', 'submitter', 'judge', 'spotlight_file', 'files',
-                  'is_last_revision', 'number', 'state', 'timeline', 'judgment_comment', 'judgment_comment_html')
+                  'is_last_revision', 'number', 'state', 'timeline', 'judgment_comment', 'judgment_comment_html',
+                  'reviewer_data')
+
+    def _get_reviewer_data(self, revision):
+        if not revision.is_last_revision:
+            return None
+
+        data = dict(revision.get_reviewer_render_data(self.context.get('user')))
+        review_type_schema = PaperReviewTypeSchema()
+        for name in {'groups', 'missing_groups', 'reviewed_groups'}:
+            data[name] = [review_type_schema.dump(item.instance) for item in data[name]]
+
+        reviews = {}
+        for key, value in data['reviews'].iteritems():
```

Comment:
I'd use `viewitems()` in new code; that can be replaced with `items()` in the (python 3) future straight away